pkg/minikube/cruntime/docker.go

```diff
@@ -314,5 +335,37 @@ func (r *Docker) Preload(k8sVersion string) error {
 	if err := r.Runner.Remove(fa); err != nil {
 		glog.Infof("error removing tarball: %v", err)
 	}
+
+	// save new reference store again
+	if err := refStore.Save(); err != nil {
+		glog.Infof("error saving reference store: %v", err)
+	}
+	// update reference store
+	if err := refStore.Update(); err != nil {
+		glog.Infof("error updating reference store: %v", err)
+	}
 	return r.Restart()
 }
+
+// ImagesPreloaded returns true if all images have been preloaded
+func ImagesPreloaded(runner command.Runner, images []string) bool {
```

Comment:
Since this is docker specific, but in the cruntime package, either add the method to the Docker struct (preferred), or name it with a Docker prefix.

Reply:
went with changing the name as the function is used by other packages as well -- I'll definitely refactor this soon when adding other container runtimes for preload